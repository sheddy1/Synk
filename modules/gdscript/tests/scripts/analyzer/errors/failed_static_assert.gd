enum MyEnum {A, B, C}
const ARRAY = [1, 2]
@static_assert(len(MyEnum) == len(ARRAY))

func test():
	pass
