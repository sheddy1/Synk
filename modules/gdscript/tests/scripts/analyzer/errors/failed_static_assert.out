GDTEST_ANALYZER_ERROR
Static assertion failed.
